README.md

```diff
@@ -390,6 +390,19 @@ Full-document verification methods allow migration-verifier to diagnose mismatch
 
 Additionally, because the amount of data sent to migration-verifier doesn’t actually reflect the documents’ size, no meaningful statistics are shown concerning the collection data size. Document counts, of course, are still shown.
 
+# Change reading methods
+
+NB: If the verifier restarts, it **MUST** use the same change reader options
+as before, or it will fail immediately.
+
+## `changeStream`
+
+The default. The verifier will read a change stream, which works seamlessly on sharded or unsharded clusters.
+
+## `tailOplog`
+
+The verifier will read the oplog continually instead of reading a change stream. This is generally faster, but it doesn’t work in sharded clusters.
```

Comment:
Should we change "doesn't work" to be "requires connecting directly to each shard"?

Reply:
They generally aren't and this is why [the directShardOperations privilege](https://www.mongodb.com/docs/manual/reference/built-in-roles/?deployment-type=self#mongodb-authrole-directShardOperations) would be needed in newer MongoDB versions. However, if we've accepted doing a shard-to-shard syncing procedure, then pairing it with a shard-to-shard verification procedure would be totally ok.